src/plugins/vue/VuePlugin.ts

```diff
@@ -0,0 +1,266 @@
+import { File } from "../../core/File";
+import { WorkFlowContext, Plugin } from "../../core/WorkflowContext";
+import { CSSPluginClass } from "../stylesheet/CSSplugin";
+import { Concat, hashString } from "../../Utils";
+import { VueBlockFile } from './VueBlockFile';
+import { VueTemplateFile } from './VueTemplateFile';
+import { VueStyleFile } from './VueStyleFile';
+import { VueScriptFile } from './VueScriptFile';
+import * as path from "path";
+import * as fs from "fs";
+import { each } from "realm-utils";
+const vueCompiler = require("vue-template-compiler");
```

Comment:
Should be hidden with the plugin. Can't be in dependencies.

Reply:
As in:

```js
transform(file) {
 const vueCompiler = require('vue-template-compiler')
}
```

Guess I'd need to update docs for dependency listing too

Reply:
That's right! The installation must include all dependencies